packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import {
+  FRIENDLY_VALUE_NOT_SET,
+  VALUE_NOT_SET,
+} from '../../../../utils/SynapseConstants'
+import { cloneDeep } from 'lodash-es'
+import EnumFacetFilterUI, { RenderedFacetValue } from './EnumFacetFilterUI'
+
+const meta = {
+  title: 'Explore/Components/Facets/EnumFacetFilter',
+  component: EnumFacetFilterUI,
```

Comment:
Note that this story now inspects the `EnumFacetFilterUI` component. This component does not rely on QueryWrapper, making it a better candidate for a story.